lib/elixir/lib/system.ex

```diff
@@ -331,8 +339,10 @@ defmodule System do
 
   The function must receive the exit status code as an argument.
   """
+  @spec at_exit((non_neg_integer -> any)) :: :ok
   def at_exit(fun) when is_function(fun, 1) do
```

Comment:
Is this a breaking change? I'm not sure if this is considered a bug.

The same applies to `ExUnit.start/1`.

Reply:
Well, technically it is a breaking change, but given that previously both had undocumented return types, it might not _really_ be considered a breaking change. If they‘re undocumented, then what gets returned is just sort of a happy accident in the eyes of many folks.